pkgs/by-name/ow/owntone/package.nix

```
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenv,
+  autoPatchelfHook,
+  autoreconfHook,
+  fetchFromGitHub,
+  nix-update-script,
+
```

OwnTune supports local playback through PulseAudio. It might be nice to add it as an option.

```suggestion

  pulseSupport ? config.pulseaudio or stdenv.hostPlatform.isLinux,

```